helm/alfresco-content-services/values.yaml

```diff
@@ -391,29 +391,123 @@ alfresco-infrastructure:
   nginx-ingress:
     enabled: false
 
-alfresco-search:
+alfrescosearch:
   enabled: true
-# If enabled is set to false, then external host and port need to point to the external instance of SOLR6, and in this case:
-# Note: Rule_05-network-policy-search will be disable.
-# Note: Rule_04-network-policy-repository internal trafic to SOLR6 instance will be disabled.
-# Note: ingress-repository will not block external trafic to */solr/*
-#  external:
-    # Host dns/ip of the external solr6 instance.
-#    host: "127.0.0.1"
-    # Port of the external solr6 instance.
-#    port: "8983"
+  # Default values for alfrescosearch.
+  # This is a YAML-formatted file.
```

Comment:
I presume these are comments from the original file, we don't need these. 

We've also lost the original comments that still appear to be useful.

Reply:
Reverted with original comments